gulp.config.js

```diff
@@ -42,6 +42,26 @@ export const settings = {
 			'Opera 12.1'
 		]
 	},
+	bootlint: {
+		stoponerror: true,
+		disabledIds: ['W005'],
+		reportFn(file, lint, isError, errorLocation) {
+			let message = (isError) ? 'ERROR! - ' : 'WARN! - ';
+			if (errorLocation) {
+				message += file.path + ' (line:' + (errorLocation.line + 1) + ', col:' + (errorLocation.column + 1) + ') [' + lint.id + '] ' + lint.message;
+			} else {
+				message += file.path + ': ' + lint.id + ' ' + lint.message;
+			}
+			console.log(message);
+		},
+		summaryReportFn(file, errorCount, warningCount) {
```

Comment:
Yes, because it doesn't produce output without it. Can you test it without on your machine?